koku/api/report/queries.py

```diff
@@ -240,16 +255,38 @@ def _check_for_operator_specific_filters(self, filter_collection):
             `.exclude`. Django adds "IS NOT NULL" when using `.exclude` which removes
             the `no-{option}` results.
         """
-        filter_collection = self._set_tag_filters(filter_collection)
-        filter_collection = self._set_operator_specified_tag_filters(filter_collection, "and")
-        filter_collection = self._set_operator_specified_tag_filters(filter_collection, "or")
-        tag_exclusion_composed = self._set_tag_exclusion_filters()
+        # Tag prefixed filters
+        tag_filters = self.get_tag_filter_keys()
+        tag_group_by = self.get_tag_group_by_keys()
+        tag_filters.extend(tag_group_by)
+        filter_collection = self._set_prefix_based_filters(filter_collection, self._mapper.tag_column, tag_filters)
+        tag_exclude_filters = self.get_tag_filter_keys("exclude")
+        tag_exclusion_composed = self._build_prefixed_based_exclusions(self._mapper.tag_column, tag_exclude_filters)
```

Comment:
Given that we call `_set_prefix_based_filters` and `_build_prefixed_based_exclusions` with _either_ tag prefixed _or_ aws prefixed lists, lets pass the prefix in, and then pass that into `strip_prefix` so we don't have to worry about updating logic in that func in the future.

Reply:
Yeah, I can put it back to the way it was 🙇